packages/state-utils/src/action-types.ts

```diff
@@ -0,0 +1,2 @@
+export const DESERIALIZE = 'DESERIALIZE' as const;
+export const SERIALIZE = 'SERIALIZE' as const;
```

Comment:
Const strings don't gain anythin with a const assertion. 

```suggestion
export const DESERIALIZE = 'DESERIALIZE';
export const SERIALIZE = 'SERIALIZE';
```

But it _looks_ like these strings aren't exported. Is that right? If so, let's use literal strings directly and remove this file.